controller/scheduler-v2/scheduler.go

```diff
@@ -18,25 +20,40 @@ import (
 	"github.com/flynn/flynn/host/types"
 	"github.com/flynn/flynn/pkg/attempt"
 	"github.com/flynn/flynn/pkg/cluster"
+	"github.com/flynn/flynn/pkg/httphelper"
 	"github.com/flynn/flynn/pkg/shutdown"
 	"github.com/flynn/flynn/pkg/status"
 	"github.com/flynn/flynn/pkg/stream"
 )
 
 const (
-	eventBufferSize int = 1000
-	serviceName         = "controller-scheduler"
+	eventBufferSize    int           = 1000
+	serviceName                      = "controller-scheduler"
+	maxJobAttempts     uint          = 30
+	jobAttemptInterval time.Duration = 500 * time.Millisecond
 )
 
 var logger = log15.New("component", "scheduler")
 
+func fnLogger(ctx ...interface{}) log15.Logger {
```

Comment:
I added this to simplify searching (both in the editor and grep). This way function names only show up once.